app/assets/javascripts/app/views/item_view.coffee

```diff
@@ -1,6 +1,63 @@
 class App.views.ItemView extends Backbone.View
-  initialize: (item) ->
-    @item = item
+  className: 'item'
+  tagName: 'li'
+  template: HandlebarsTemplates['item']
+
+  initialize: (options) ->
+    @item = options.item
+    @children = new App.collections.Items(@item.get('children'))
+    @$el.attr('id', @item.id)
+    @showChildren = true
+
+  events:
+    'click .toggle-children': 'toggleChildren'
+    'click h2': 'toggleh2Editable'
+    'click p': 'togglepEditable'
+
+  toggleh2Editable: (e) ->
+    e.stopPropagation()
+    $(@$el.find('h2.uneditable')[0]).addClass('hide-edit')
+    $(@$el.find('h2.editable')[0]).removeClass('hide-edit')
+
+  togglepEditable: (e) ->
+    e.stopPropagation()
+    $(@$el.find('p.uneditable')[0]).addClass('hide-edit')
+    $(@$el.find('p.editable')[0]).removeClass('hide-edit')
+
+  toggleChildren: (e) ->
+    e.stopPropagation()
+    @showChildren = !@showChildren
+    if @showChildren then toggleSwitch = '[-]' else toggleSwitch = '[+]'
+
+    $(e.currentTarget).html(toggleSwitch)
+    nextol = $($(e.delegateTarget).find('ol')[0])
```

Comment:
there are better idioms to select out the first of a selector result set, provided by both CSS and jQuery